draft-ietf-scitt-architecture.md

```diff
@@ -597,13 +596,13 @@ In essence, when using X.509 Signed Statements, the Transparency Service MUST bu
 The protected header of the COSE_Sign1 Envelope MUST include either the Issuer's certificate as `x5t` or the chain including the Issuer's certificate as `x5chain`.
 If `x5t` is included in the protected header, an `x5chain` with a leaf certificate corresponding to the `x5t` value MAY be included in the unprotected header.
 
-The Transparency Service MUST apply the Registration Policy that was most recently added to the Append-only Log at the time of Registration.
+The Transparency Service MUST apply the Registration Policy that was most recently added to the Verifiable Data Structure at the time of Registration.
```

Comment:
```suggestion
The Transparency Service MUST apply the Registration Policy that was most recently added to the Append-only Log at the time of Registration.
```

The VDS records only digests of entries in the log, it will contain a commitment to, but not the policy itself.

Reply:
Merkle trees only store digests... other stuff might store the full signed statement.
Don't assume merkle trees would be my recommendation.
The important part is that the client accepts the security mechanism used by the TS... submission to a TS implies the desire to have a receipt produced from the VDS... regardless of if the VDS is a log or not.

Reply:
If we want to restrict the architecture to specific VDS (CT / CCF logs), that seems like a better solution than using the word "append only log".

Reply:
I've re-written the suggestion to use VDS, but use the word "commit" rather than "add". The VDS produces commitments to entries, but not entries themselves.

I don't know what the other stuff is, CT, CCF and the MMR all use Merkle Trees. The only "example" ever mentioned in the COMETRE spec is Merkle Trees also. I am always happy to learn, but at the moment I do not believe that confusing the VDS and the log of full entries is net positive.

Reply:
@achamayou there exist production systems which use mathematical objects, rather than hashes, for the commitment scheme. "trees of math" can be built out of these just fine and can offer advantages in size and privacy. 

A more readily comparible approach to merkle trees is this one - https://math.mit.edu/research/highschool/primes/materials/2018/Kuszmaul.pdf 

The minima blockchain uses a clever recursive proof to provide a constant size commitment to the entire state of their chain.

There is quite a lot of overlap with the verifiable compute and zk spaces. Nova and Halo. See this eprint for a starting example. https://eprint.iacr.org/2023/969.pdf 

The intent has always been to avoid excluding these sorts of things, and we prototyped verkle's in 2023.